src/build-tasks/build-configuration.ts

```diff
@@ -224,6 +224,8 @@ export class BuildConfiguration {
         this.resolver.addMappings(this.mappings);
         this.resolver.setFilePath(filePath);
         this.resolver.addParameter('AWS::Partition', AwsUtil.partition);
+        this.resolver.addParameter('IsPartition', AwsUtil.GetIsPartition());
```

Comment:
@brcourt should would rename these parameters to something like:
ORG::IsCommercial
ORG::IsPartition

Reply:
Yeah thats probably a good idea.